view/frontend/web/js/store/index.js

```diff
@@ -36,7 +36,44 @@ const store = new Vuex.Store({
     },
     paymentMethods: [],
     shippingMethods: [],
-    shippingInformation: shippingInformation,
+    shippingInformation: {
+      "addressInformation": {
+        "shipping_address": {
+            "region": "string",
+            "region_id": 0,
+            "country_id": "string",
+            "street": [
+                "string"
+            ],
+            "company": "string",
+            "telephone": "string",
+            "postcode": "string",
+            "city": "string",
+            "firstname": "string",
+            "lastname": "string",
+            "email": "string",
+            "same_as_billing": 0
+        },
+        "billing_address": {
+            "region": "string",
+            "region_id": 0,
+            "region_code": "string",
+            "country_id": "string",
+            "street": [
+                "string"
+            ],
+            "company": "string",
+            "telephone": "string",
+            "postcode": "string",
+            "city": "string",
+            "firstname": "string",
+            "lastname": "string",
+            "email": "string"
+        },
+        "shipping_method_code": "string",
+        "shipping_carrier_code": "string"
+    }
```

Comment:
please delete this file b/c I see it's the same config as we have in `/templates/data/shipping-information.phtml`

Reply:
ok, I deleted it